bpy-build.yaml

```
@@ -1,22 +1,13 @@
 addon_folder: MCprep_addon
 build_name: MCprep_addon
 
+build_extension: true
+extension_settings:
+  build_legacy: true
+
 install_versions:
-  - 4.3
-  - 4.2
-  - 4.1
-  - 4.0
-  - 3.6 
-  - 3.5
-  - 3.4
-  - 3.3
-  - 3.2
-  - 3.1
-  - 3.0
-  - 2.93
-  - 2.9
-  - 2.83
-  - 2.8
+  - 4.3+
+  - 2.8..4.3
```

Is it possible to do this or no?

```suggestion
  - 2.83..4.3
```

---

Yep, that's possible as well, though are we dropping 2.8 to 2.82 support?